README.md

```diff
@@ -54,3 +54,30 @@ A fixer corresponds to a particular test.
 
 For example, if `Files encoding` test from [Meta.Tests.ps1](Meta.Tests.ps1) test fails, 
 you should be able to run `ConvertTo-UTF8` fixer from [MetaFixers.psm1](MetaFixers.psm1).
+
+## TestHelper
+The test helper module contains cmdlets for assiting in either running standard DSC
+Resource Pester tests or creating a NuGet package from the build.
+
+## Example Usage in AppVeyor.yml
+To automatically download and install the DscResource.Tests in an AppVeyor.yml file, add
+the following to the install section:
+
+```
```

Comment:
I'd recommend linking to the appveyor.yml example here instead of copying the snippets to get rid of redunancy: https://github.com/PowerShell/DscResources/blob/master/DscResource.Template/appveyor.yml 

It contains the same code, but with broader context, so let's keep it in one place.